developer_tools/malloc_dbg.c

```diff
@@ -0,0 +1,252 @@
+/**
+ * @file
+ * @brief This file contains malloc_dbg, calloc_dbg, free_dbg and printLeaks implementations.
+ * @author tinouduart33

```

Comment:
In the auto-generated documentation website, we can use Markdown here. 🙂
```suggestion
 * @author [tinouduart33](https://github.com/tinouduart33)
```